_drafts/http4s-tutorial.md

```diff
@@ -0,0 +1,887 @@
+---
+title: "Unleashing the Power of HTTP Apis: The Http4s Library"
+date: 2021-05-07 
```

Comment:
Let's just change the date here and we'll be good to merge. I'll shoot the video early next week

Reply:
Ok. Which date?

Reply:
June 7? (and publish tomorrow)

Reply:
Done. Remember to move the file from the `_draft` folder to the `_post` folder :)